### DISCUSSION TITLE
Bitwarden: Allow OTP's copy to clipboard

### DISCUSSION THREAD

**Discussion by FMGordillo** [minbrowser/min#2085] [Ideas]
Something that stops me from daily driving this web browser is the ability to, when I autocomplete a login page, my OTP is copied to my clipboard and I can paste if effortlessly.

**Comment by sgtnasty** [comment]
As I understand, *min* uses the [Bitwarden CLU](https://bitwarden.com/help/cli/) which does not appear to support getting OPT codes from the vault.
